.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 cache:
   directories:
     - $GOCACHE
-    - $GOPATH
```

Comment:
@torkelrogstad Sorry, this may look out of context because of rebasing. 

Looks like CI has cached my initial version with a failing test and can't invalidate the cache even after providing a code with sucessful tests, e.g.:

https://travis-ci.org/btcsuite/btcd/builds/647711967
https://travis-ci.org/btcsuite/btcd/builds/647712943
https://travis-ci.org/btcsuite/btcd/builds/647769818

These builds worked both locally & on CI builds for my personal fork:
https://github.com/pavel-main/btcd/pull/1

@jakesyl Was also encountering this issue before in previous PR:
https://github.com/btcsuite/btcd/pull/1503#issuecomment-583759870

Completely understand if you wish to keep this line for now - I can just try pushing a new branch & opening new PR from scratch

Reply:
Hm, I see. I don't really have super strong feelings on this, as I don't have much knowledge of the CI setup here.

Reply:
@torkelrogstad Feels like a Travis CI bug, exactly or similar to this one - https://github.com/travis-ci/travis-ci/issues/3055#issuecomment-127416882

They're storing cached artifacts elsewhere (probably on S3), so something might got broken there. Perhaps a quicker and cleaner workaround is to [clear caches on Travis side](https://docs.travis-ci.com/user/caching/#clearing-caches) once and then we'll rollback `.travis.yml` changes to try again

Reply:
If this is flaking tests, let's just remove it

Reply:
See #1537